python/lsst/ts/standardscripts/scheduler/test_utils/scheduler_controller.py

```diff
@@ -0,0 +1,140 @@
+# This file is part of ts_standardscripts
```

Comment:
I *very strongly* suggest renaming this directory to `testutils`. I've found that pytest tries to run things in any module named `test_foo`, which is a real nuisance.